test/src/module/common/ioTlsTest.c

```diff
@@ -949,5 +950,52 @@ testRun(void)
         TEST_RESULT_PTR_NE(statToJson(), NULL, "check");
     }
 
+    // *****************************************************************************************************************************
+    if (testBegin("harnessIo"))
```

Comment:
We don't have any other tests names 'harness ...' instead we talk about testing the function and we may use a harness/shim for that, right?  Maybe this should be named differently?

Reply:
will update to (testBegin("verify shimmed ioSessionFd()"))